packages/request-pre/src/config.ts

```
@@ -0,0 +1,79 @@
+            resolve,
+            reject,
+        }; 
+    });
+}
+export default {
+    async pre(config: ApiSchemaConfig, configMap: apiConfig, initData: any, ctx: ApiSchema): Promise<any> {
+        if (config) {
+            return getConfig(config, configMap, 50).reduce((pre: Promise<any>, configHandler: ConfigHandler): Promise<any> => {
+                return pre.then((prevData: any): void => {
+                    if (configHandler.resolve) {
+                        return configHandler.resolve(ctx, configHandler.params, prevData);
```

Biome

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)